src/Expensify.js

```diff
@@ -196,10 +196,10 @@ function Expensify(props) {
         <DeeplinkWrapper isAuthenticated={isAuthenticated}>
             {shouldInit && (
                 <>
+                    <DownloadAppModal />
```

Comment:
Could we just pass `isAuthenticated` in here to avoid code duplication?

Reply:
Pushed this change